DuckDuckGo/DaxDialogs.swift

```diff
@@ -163,6 +163,9 @@ final class DaxDialogs {
     private let variantManager: VariantManager
 
     private var nextHomeScreenMessageOverride: HomeScreenSpec?
+    
+    // So we can avoid showing two dialogs for the same page
+    private var lastURLDaxDialogReturnedFor: URL?
```

Comment:
I did consider storing this info on the tab, since then we wouldn't need to worry about clearing it with the fire button, but ultimately I think it would be the wrong place for any of this logic to live and compromise testability